### PR TITLE
Astro 3/1/23

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -23,6 +23,23 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+    - `AIRFLOW__METRICS__STATSD_ON` 
+    - `AIRFLOW__METRICS__STATSD_HOST`
+    - `AIRFLOW__METRICS__STATSD_PORT`
+    - `AIRFLOW__METRICS__STATSD_ALLOW_LIST` 
+    - `AIRFLOW__METRICS__STATSD_STATSD_CUSTOM_CLIENT_PATH`
+    - `AIRFLOW__METRICS__STATSD_PREFIX`
+
+    You can still set new values for these variables, but those values are automatically overwritten in the Astro data plane. See [Platform variables](platform-variables.md).
+  
+- Fixed an issue where a user could be provisioned multiple accounts if they logged in using differently cased characters in their login email address.
```
Comment (kwcanuck) [inline astro/release-notes.md:41]:
```suggestion
- Fixed an issue where a user could provision multiple accounts when their login email address included differently cased characters.
```

File: astro/release-notes.md [hunk 2]
```diff
@@ -23,6 +23,23 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.11.0 ([Release notes](cli/release-notes.md))
 
+## February 28, 2023
+
+### Bug fixes
+
+- To protect Astro's monitoring functionality, you can no longer override the values of the following environment variables:
```
Comment (kwcanuck) [inline astro/release-notes.md:30]:
```suggestion
- To ensure the reliability of Astro monitoring, you can no longer override the values of the following environment variables:
```

**Review by kwcanuck — APPROVED** [review]

**Comment by chomipi88** [comment]
As part of this release we are locking down the GKE and AKS control plane authorized network to the Astro Control Plane.

https://github.com/astronomer/astro/issues/1357
https://github.com/astronomer/astro/issues/3884

File: astro/release-notes.md [hunk 3]
```diff
@@ -23,6 +23,33 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+### Access Astronomer through the Okta dashboard
+
+If your Organization uses Okta as your Astro identity provider, you can now log in to Astro directly from your [Okta apps dashboard](https://help.okta.com/eu/en-us/Content/Topics/end-user/dashboard-overview.htm). If you're already authenticated through Okta, you no longer have to reauthenticate in Astro if you access it through your dashboard.
+
+If you want users to always authenticate through Astro, you can disable this feature. See [Restrict identity provider initiated logins](manage-organization.md#restrict-identity-provider-initiated-logins).
+
+### Additional improvements
+
+- Ingress to GCP and Azure clusters is now limited to control plane IPs. This change will be implemented on all clusters in the coming weeks.
```
Comment (kwcanuck) [inline astro/release-notes.md:36]:
```suggestion
- Access to Google Cloud Platform (GCP) and Azure clusters is now limited to control plane IPs. This change will be implemented on all clusters in the coming weeks.
```

File: astro/release-notes.md [hunk 4]
```diff
@@ -23,6 +23,33 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.11.0 ([Release notes](cli/release-notes.md))
 
+## February 28, 2023
+
+### Access Astronomer through the Okta dashboard
+
+If your Organization uses Okta as your Astro identity provider, you can now log in to Astro directly from your [Okta apps dashboard](https://help.okta.com/eu/en-us/Content/Topics/end-user/dashboard-overview.htm). If you're already authenticated through Okta, you no longer have to reauthenticate in Astro if you access it through your dashboard.
```
Comment (kwcanuck) [inline astro/release-notes.md:30]:
```suggestion
If your Organization uses Okta as your Astro identity provider, you can now log in to Astro directly from your [Okta Apps dashboard](https://help.okta.com/eu/en-us/Content/Topics/end-user/dashboard-overview.htm). If you've already been authenticated by Okta, you no longer have to be authenticated by Astro when you access it through your dashboard.
```

File: astro/release-notes.md [hunk 5]
```diff
@@ -162,7 +189,7 @@ See [Introducing Astro’s New Workspace Homepage](https://www.astronomer.io/blo
 
 ### Additional improvements
 
-- Data plane cluster access is now limited to control plane IPs. This change will be implemented on all clusters in the coming weeks.
+- Ingress to AWS clusters is now limited to control plane IPs. This change will be implemented on all clusters in the coming weeks.
```
Comment (jwitz) [inline astro/release-notes.md:192]:
```suggestion
- Ingress to the Airflow UI and API on Astro clusters is now limited to control plane IPs. This change will be implemented on all clusters in the coming weeks.
```

File: astro/release-notes.md [hunk 6]
```diff
@@ -23,6 +23,33 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.11.0 ([Release notes](cli/release-notes.md))
 
+## February 28, 2023
```
Comment (jwitz) [inline astro/release-notes.md:26]:
```suggestion
## March 1, 2023
```

File: astro/release-notes.md [hunk 7]
```diff
@@ -23,6 +23,33 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+- To protect the functionality of Astro monitoring services, you can no longer override the values of the following environment variables:
+  
+    - `AIRFLOW__METRICS__STATSD_ON` 
+    - `AIRFLOW__METRICS__STATSD_HOST`
+    - `AIRFLOW__METRICS__STATSD_PORT`
+    - `AIRFLOW__METRICS__STATSD_ALLOW_LIST` 
+    - `AIRFLOW__METRICS__STATSD_STATSD_CUSTOM_CLIENT_PATH`
+    - `AIRFLOW__METRICS__STATSD_PREFIX`
+
+    You can still set new values for these variables, but those values are automatically overwritten in the Astro data plane. See [Platform variables](platform-variables.md).
```
Comment (jwitz) [inline astro/release-notes.md:49]:
```suggestion
    You can still set new values for these variables, but the values will be automatically overwritten in the Astro data plane. See [Platform variables](platform-variables.md).
```

**Pull request merged**